src/contexts/APIContext/APIContext.js

```diff
@@ -0,0 +1,54 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import axios from "axios"
+
+export const getProducts = async () => {
+    const res = await axios.get("/api/products");
+    const products = await res.data.products;
+    return products;
+}
+
+export const getCategories = async () => {
+    const res = await axios.get("/api/categories");
+    const categories = await res.data.categories;
+    return categories;
+}
+
+const APIContext = createContext(null)
+
+const useAPI = () => useContext(APIContext)
+
```

Comment:
useAPI does not convey exactly what the context is doing, may rename it if possible

Reply:
Sure, Thanks